src_assets/linux/misc/firewall_rules.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<service>
+  <short>@PROJECT_NAME@</short>
+  <description>@PROJECT_DESCRIPTION@</description>
+  <port protocol="udp" port="29922"/>
+  <port protocol="tcp" port="47984"/>
+  <port protocol="tcp" port="47989"/>
+  <port protocol="udp" port="47998"/>
+  <port protocol="udp" port="47999"/>
+  <port protocol="udp" port="48000"/>
+  <port protocol="tcp" port="48010"/>
```

Comment:
Thats the ports that my router opend as upnp. I can change the ports if the are not correct.
But thats a problem if somebody changes the ports :/

Reply:
you can view the current rules with

sudo firewall-cmd --list-all-zones

Reply:
sudo firewall-cmd --permanent --zone=public --remove-service=sunshine
sudo firewall-cmd --reload

Reply:
Okay, you can add this to the docs.

`./docs/source/about/usage.rst`

L121
```rst
#. Optionally, configure firewall
      .. Note:: This is not currently available for AppImage or Flatpak.

      .. Warning:: This will not work if you change the family of ports.

      Enable
         .. code-block:: bash

            sudo firewall-cmd --permanent --zone=public --add-service=sunshine
            sudo firewall-cmd --reload

      Disable
         .. code-block:: bash

            sudo firewall-cmd --permanent --zone=public --remove-service=sunshine
            sudo firewall-cmd --reload
```

Reply:
Done <3